### PR TITLE
Fix validation messages, adjust new org and new project wording

### PR DESCRIPTION
**Pull request by timhaines**
Validation messages said "Name must be blank" when it was already blank. This is due to a bug with ember-validators: https://github.com/offirgolan/ember-validators

Also adjusted the wording on new org and new project screens to be a little friendlier.

### CODE REVIEW

File: app/validations/organization-edit.js [hunk 1]
```diff
@@ -4,9 +4,9 @@ import {
 
 export default {
   name: [
-    validatePresence(true),
+    validatePresence({presence: true, message: 'Name can\'t be blank'}),
```
Comment (fotinakis) [inline app/validations/organization-edit.js]: Style: prefer double quotes around the string if you're going to use an apostrophe or string interpolation inside. Alternatively, use the new ES6 `` quoted string.

File: app/templates/organization/index.hbs [hunk 2]
```diff
@@ -25,12 +25,15 @@
...
+          <p>
+            After you've created a project, you can use our demo apps for experimentation.
+          </p>
           {{!-- TODO: #billingadmins --}}
           {{#if (eq "admin" model.currentUserMembership.role)}}
             <p>
               You can also
               {{#link-to "organizations.organization.users.invite" model.slug}}invite team members{{/link-to}}
-              to join.
+              to join from the settings page.
```
Comment (fotinakis) [inline app/templates/organization/index.hbs]: Feels redundant to add this if it's already linked up.

Comment (timhaines) [inline app/templates/organization/index.hbs]: @fotinakis I added it as a reference of where to find it later, if they don't happen to click the link now.  Happy to remove if you feel med/strong about it.

**Comment by timhaines** [comment]
Needs simplifying to remove the custom error messages.  The underlying package has been fixed now.

**Pull request merged**